packages/compass-sidebar/src/components/connection-info-modal.tsx

```diff
@@ -46,6 +48,7 @@ export function ConnectionInfoModal({
   close,
   infos,
 }: {
+  initialConnectionInfo: ConnectionInfo;
```

Comment:
this file has it's own `ConnectionInfo`, but in this prop we send the original - `ConnectionStorageConnectionInfo`. this is what is forcing the "as any" in the sidebar

Reply:
I can fix this in my PR, just saying :)

Reply:
Ah yeah I can fix it, you are right. It takes a minute. Thanks for pointing out!